ada_feeding/ada_feeding/trees/acquire_food_tree.py

```diff
@@ -6,96 +6,65 @@
-    # of its constraints.
+    Tree Blackboard Inputs:
+    - camera_info: See ComputeFoodFrame
+    - mask: See ComputeFoodFrame
+
+    Tree Blackboard Outputs:
+
+    """
 
-    # pylint: disable=too-many-locals
-    # Unfortunately, many local variables are required here to isolate the keys
-    # of similar constraints in the blackboard.
-    def create_move_to_tree(
+    @override
+    def create_tree(
         self,
         name: str,
-        tree_root_name: str,
-        node: Node,
+        tree_root_name: str,  # DEPRECATED
     ) -> py_trees.trees.BehaviourTree:
-        """
-        Creates the AcquireFood behavior tree.
-
-        Parameters
-        ----------
-        name: The name of the behavior tree. (DEPRECATED)
-        tree_root_name: The name of the tree. This is necessary because sometimes
-            trees create subtrees, but still need to track the top-level tree
-            name to read/write the correct blackboard variables. (DEPRECATED)
-        node: The ROS2 node that this tree is associated with. Necessary for
-            behaviors within the tree connect to ROS topics/services/actions.
-
-        Returns
-        -------
-        tree: The behavior tree that acquires food
-        """
-        # TODO: remove name and tree_root_name
-        # We have access to them implicitly via self.blackboard / self.blackboard_tree_root
-
-        ### Inputs we expect on the blackboard on initialization
-        # Note: WRITE access since send_goal could write to these
-        # Mask for ComputeFoodFrame
-        self.blackboard.register_key(key="mask", access=py_trees.common.Access.WRITE)
-        # CameraInfo for ComputeFoodFrame
-        self.blackboard.register_key(
-            key="camera_info", access=py_trees.common.Access.WRITE
-        )
+        # Docstring copied by @override
 
-        ### Define Tree Leaves and Subtrees
-
-        # Add ComputeFoodFrame
-        compute_food_frame = ComputeFoodFrame(
-            "ComputeFoodFrame",
-            self.blackboard.namespace,
-            inputs={
-                "ros2_node": node,
-                "camera_info": BlackboardKey("camera_info"),
-                "mask": BlackboardKey("mask")
-                # Default world_frame = "world"
-                # Default debug_tf_frame = "food"
-            },
-            outputs={
-                "action_select_request": None,
-                "food_frame": None,
-                "debug_tf_publisher": BlackboardKey("debug_tf_publisher")
-                # ^^ Set to None to disable
-            },
-        )
+        # TODO: remove tree_root_name
+        # Sub-trees in general should not need knowledge of their parent.
 
         ### Define Tree Logic
 
         # Root Sequence
         root_seq = py_trees.composites.Sequence(
-            name="RootSequence",
+            name=name,
             memory=True,
             children=[
-                compute_food_frame,
+                py_trees.decorators.Timeout(
```

Comment:
I feel like the default duration of 5secs is probably too long to wait. What do you think about 1s?